docs/ha-considerations.md

```diff
@@ -0,0 +1,332 @@
+# High Availability Considerations
+
```

Comment:
I think we have to add a disclaimer on top of this documents stating that:
- this page contains a collection of community provided High Availability Considerations
- if a user find something incomplete, not clear, or if the user is willing to document alternative solutions, contributions are well accepted.
- being this a community provided list of consderations, the best place to ask support or information about this page is the kubeadm channel where most of the contributors are usually active

@neolit123 something else to add to clarify "the contract/the support matrix" around this page?

Reply:
Thanks, I have created a disclaimer with what you proposed.

Reply:
> @neolit123 something else to add to clarify "the contract/the support matrix" around this page?

not sure what you mean?

Reply:
@neolit123 I was asking if you want to add/change something in the disclaimer

Reply:
ok, the disclaimer seems fine for now.